### PR TITLE
[SHARE-716][Task] Trim and add licenses to requirements.txt

### PR DESCRIPTION
**Pull request by chrisseto**

### CODE REVIEW

**Comment by leb2dg** [comment]
@chrisseto, a few elasticsearch tests are failing on this

**Comment by chrisseto** [comment]
@leb2dg I think develop might be failing right now.

**Comment by leb2dg** [comment]
:(

File: requirements.txt [hunk 1]
```diff
@@ -1,58 +1,53 @@
...
-django==1.9.7
-django[bcrypt]==1.9.7
-django-celery==3.1.17
-django-extensions==1.6.7
-django-allauth==0.31.0
-django-guardian==1.4.4
-django-oauth-toolkit==0.10.0
-django-revproxy==0.9.12
-django-mptt==0.8.5
+# newrelic APM agent
```
Comment (aaxelb) [inline requirements.txt]: This shouldn't be sorted first

**Pull request merged**